packages/sql/src/Client.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * @since 1.0.0
+ * @since 2.0.0
```

Comment:
You can leave this as `1.0.0`, as this isn't part of the `effect` package.

Reply:
done 👍